src/details/ast.cpp

```diff
@@ -760,34 +737,36 @@ GlobalVariable::GlobalVariable(ts::Node node) : g_var(node) {
         throw runtime_error("not a global variable node");
     }
 }
-auto GlobalVariable::type() -> GV {
-    if (g_var.text() == "_G") {
+auto GlobalVariable::type() const -> GV {
+    if (this->g_var.text() == "_G") {
         return GV::_G;
     } else {
         return GV::_VERSION;
     }
 }
```

Comment:
I think we can also remove `GlobalVariable`. They are (kind of) special variables but act like normal variables.